url_screenshot/plugin.spec.yaml

```
@@ -0,0 +1,50 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: url_screenshot
+title: Screenshot a URL
+description: Screenshot a URL from the Orchestrator
```

```suggestion
description: Screenshot a URL
```
I'm not sure if the orchestrator part is necessary, it may be too technical for the description.

---

I feel it needs to be in there someplace since it is significantly different behavior from other screenshot plugins  where they screenshot from someplace out on the internet.  Do you have any suggestions for better wording?